src/main/java/cookbuddy/logic/parser/ViewCommandParser.java

```diff
@@ -25,7 +24,7 @@ public ViewCommand parse(String args) throws ParseException {
             return new ViewCommand(index);
         } catch (ParseException pe) {
             throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewCommand.MESSAGE_USAGE), pe);
+                String.format(MESSAGE_INVALID_RECIPE_DISPLAYED_INDEX, ViewCommand.MESSAGE_USAGE), pe);
```

Comment:
This will just cause a different error message to be thrown.

Reply:
If an invalid command is given, it will still throw an invalid index error.

Reply:
Oh yea lemme change it